src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs

```diff
@@ -125,6 +125,21 @@ public StackTrace(StackFrame frame)
             _numOfFrames = 1;
         }
 
+        /// <summary>
+        /// Constructs a "fake" stack trace
+        /// </summary>
```

Comment:
Please provide a better description as this will end up in the public documentation. Also, provide the param element describing `frames`

Reply:
Updated